Ghidra/Features/Decompiler/src/decompile/cpp/semantics.hh

```diff
@@ -46,7 +46,7 @@ private:
   static void printHandleSelector(ostream &s,v_field val);
   static v_field readHandleSelector(const string &name);
 public:
-  ConstTpl(void) { type = real; value_real = 0; }
+  ConstTpl(void) { type = real; value_real = 0; select = v_space; }
```

Comment:
`select` is left uninitialised. We then invoke a copy constructor and copy the uninitialised value to a new `ConstTpl` and explicitly give it a value that is invalid for the enumeration.

Reply:
wonder if the should be an `v_invalid` for this enum and init to that

Reply:
Seems reasonable. What do you think @ryanmkurtz?

Reply:
I'm going to just keep it this way for consistency for the time being. I can follow up with another change to add "invalid" kinds over the codebase.